lib/ethereum/client.rb

```diff
@@ -28,6 +28,7 @@ def initialize(log = false)
     def self.create(host_or_ipcpath, log = false)
       return IpcClient.new(host_or_ipcpath, log) if host_or_ipcpath.end_with? '.ipc'
       return HttpClient.new(host_or_ipcpath, log) if host_or_ipcpath.start_with? 'http'
+      return WssClient.new(host_or_ipcpath, log) if host_or_ipcpath.start_with? 'wss'
```

Comment:
It should also support insecure websockets.

```suggestion
      return WsClient.new(host_or_ipcpath, log) if host_or_ipcpath.start_with? 'ws'
```